### PR TITLE
code chef challenge problems

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** #3074 
**Changes:**
[Add here what changes were made in this pull request.]

1. Added _Add Two Numbers in cpp_
2. Added _Find Remainder in cpp_
3. Added _First and Last Digit in cpp_

### CODE REVIEW

**Comment by tbhaxor** [comment]
@AdiChat i will commit changes in this branch only

**Comment by arnavb** [comment]
Sorry, approved twice!

**Comment by AdiChat** [comment]
We must have separate folders for each problem with a readme with problem statement and link to the original problem.

For instance, we must have a folder `FLOW001` with the file `FLOW001.cpp` and a `readme.md` with the description and link. 

Kindly reflect this change. 👍

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// part of cosmos from opengenus foundation
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Not necessary

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+// problem code link https://www.codechef.com/problems/FLOW001
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Create a readme instead. Look at the project Euler problems as examples.

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+
+#include <iostream>
+using namespace std;
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: std:: prefix everything, not this.

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+int main()
+{
+    int t;
+    cin>>t;
+    int arr[t], i=0;
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Vectors, vectors, vectors!

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    while(t--)
+    {
+        int a,b;
+	cin>>a>>b;
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Fix indetation

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp (hunk 4 above)
Comment (arnavb) [reply code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Separate lines

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+// problem code link https://www.codechef.com/problems/FLOW001
+
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int t;
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW001.cpp]: Better names, and add a prompt

File: code/online_challenges/code_chef/Beginner/FLOW001.cpp (hunk 1 above)
Comment (arnavb) [reply code/online_challenges/code_chef/Beginner/FLOW001.cpp]: The folder name should be beginner, not Beginner

File: code/online_challenges/code_chef/Beginner/FLOW002.cpp [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// part of cosmos from opengenus foundation
+
+// problem code link https://www.codechef.com/problems/FLOW002
+
+
+#include <iostream>
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW002.cpp]: Same problems as previous file.

File: code/online_challenges/code_chef/Beginner/FLOW004.cpp [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+   return firstDigit + lastDigit; 
+}
+
+int main()
+{
+    int t;
+    cin>>t;
+    int arr[t], i=0;
+    while(t--)
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW004.cpp]: Same issues for main as previous files.

File: code/online_challenges/code_chef/Beginner/FLOW006.cpp [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        number /= 10;
+    }
+
+    return sum; 
+}
+
+int main()
+{
+    int t;
+    cin>>t;
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW006.cpp]: Here too

File: code/online_challenges/code_chef/Beginner/FLOW006.cpp [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+
+#include <iostream>
+using namespace std;
+
+int getSum(int number)
+{
+    int sum = 0;
+
+    while(number != 0)
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW006.cpp]: Add spacing

File: code/online_challenges/code_chef/Beginner/FLOW007.cpp [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+    return rev; 
+}
+
+int main()
+{
+    int t;
+    cin>>t;
+    int arr[t], i=0;
+    while(t--)
```
Comment (arnavb) [inline code/online_challenges/code_chef/Beginner/FLOW007.cpp]: Here too

**Comment by tbhaxor** [comment]
@AdiChat @arnavb i am busy coz of my exams. So i will do these changes after exams.

**Comment by arnavb** [comment]
Ok, take your time! Everyone knows how tedious exams can be! 😂😭

**Pull request closed without merging**